examples/website/image-tile/README.md

```diff
@@ -0,0 +1,19 @@
+This is a standalone high resolution image example using TileLayer and BitmapLayer, with non-geospatial coordinates
```

Comment:
It would be very helpful to other users if you also explain how the tiles are generated.

Reply:
Addressed below.